azure/defaults.go

```diff
@@ -300,12 +300,13 @@ func GetBootstrappingVMExtension(osType string, cloud string, vmName string, cpu
 	// currently, the bootstrap extension is only available in AzurePublicCloud.
 	if osType == LinuxOS && cloud == PublicCloudName {
 		// The command checks for the existence of the bootstrapSentinelFile on the machine, with retries and sleep between retries.
-		// We set the version to 1.1.1 for arm64 machines and 1.0 for x64. This is due to a known issue with newer versions of
+		// We set the version to 1.1 (will target 1.1.1) for arm64 machines and 1.0 for x64. This is due to a known issue with newer versions of
 		// Go on Ubuntu 20.04. The issue is being tracked here: https://github.com/golang/go/issues/58550
 		// TODO: Remove this once the issue is fixed, or when Ubuntu 20.04 is no longer supported.
+                // We are using 1.1 instead of 1.1.1 for Arm64 as AzureAPI do not allow us to specify the full version.
```

Comment:
there's a format error here being returned by the linter 

```
azure/defaults.go:306: File is not `gci`-ed with --skip-generated -s standard,default (gci)
                // We are using 1.1 instead of 1.1.1 for Arm64 as AzureAPI do not allow us to specify the full version.
```